server/route.js

```diff
@@ -164,7 +154,7 @@ var serverRoutes =  function (router) {
 	});
 
 	router.addRoute('/invoices/edit/:deletedInvoices?', function (req, res, match){
-	  	require('./handlers/edit-invoices.js')(req, res);
+	  	require('./handlers/edit-invoices-reminders.js')(req, res, 'invoice');
```

Comment:
In the file we still refer to all the variables as if everything is an invoice. If we are also using the same handler for reminders can we write the code to be more generic? Just better naming.